tests/queries/0_stateless/02344_show_caches.reference

```diff
@@ -1,14 +0,0 @@
-cached_azure
```

Comment:
removed this test because it is impossible to make it not flaky with custom nested disk configuration which I used in test tests/queries/0_stateless/02714_local_object_storage.sql